### PR TITLE
Matthew/admin upload starter kit fix

### PR DESCRIPTION
**Pull request by MTSchoeman**
- Added Document TestData 
- Updated Unit Tests
- Made changes to Service and Controller for Documents
controller unit test
controller

### CODE REVIEW

**Review by dvanhout89 — APPROVED** [review]
👍🏻

File: RGO.Services.Tests/Services/EmployeeDocumentServiceUnitTest.cs [hunk 1]
```diff
@@ -5,97 +5,153 @@
...
+            .Returns(
+                new List<EmployeeBanking>
+                {
+                    new(EmployeeBankingTestData.EmployeeBankingDto)
+                }.AsQueryable().BuildMock());
+
+
+        List<EmployeeRole> empRoles = new List<EmployeeRole> {
+            new EmployeeRole(
+                new EmployeeRoleDto(1, EmployeeTestData.EmployeeDto, new RoleDto(3, "Employee"))
```
Comment (AWallsRR) [inline RGO.Services.Tests/Services/EmployeeDocumentServiceUnitTest.cs:64]: RoleDto(3, "Employee")

Should we not also add these to the test data?

**Review by LeratoMalinga — APPROVED** [review]

File: RGO.Services.Tests/Services/EmployeeDocumentServiceUnitTest.cs (hunk 1 above)
Comment (MTSchoeman) [reply RGO.Services.Tests/Services/EmployeeDocumentServiceUnitTest.cs:64]: I looked over it, but it would make sense to

**Review by AWallsRR — APPROVED** [review]
Nice

**Review by DewanAlberts — APPROVED** [review]

**Pull request merged**